src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -653,5 +653,34 @@
   </main>
 </div>
 
+<!-- TEMPORARY -->
+<!-- =============== -->
+<script>
+  const checkCLPAuth = () => {
+    const password = prompt('This page is in BETA. Please enter the password to access it:');
```

Comment:
Silly silly suggestion but can we use `passphrase` instead of `password`? I heard something a long time ago about special things around the use of the word "password" lol

Reply:
Not a problem, updated 👍